### DISCUSSION TITLE
编译错误

### DISCUSSION THREAD

**Discussion by Faker0715** [chenshuo/muduo#646] [General]
![image](https://user-images.githubusercontent.com/36560441/185358492-173cef99-c53f-4dcd-928e-a7de3ddc5a8c.png)

**Comment by chenshuo** [comment]
请遵循提问模板：

Please specify the exact environment where the issue occurs:

## Linux distro and version? x86 or ARM? 32-bit or 64-bit?

## Branch (cpp98/cpp11/cpp17) and version of muduo?

## Version of cmake, gcc and boost? (If not from distro.)

**Comment by chenshuo** [comment]
Fixed by #649.
